snippets/operator-group-unique-name.adoc

```diff
@@ -0,0 +1,25 @@
+// Text snippet included in the following assemblies:
+//
+// *
+//
+// Text snippet included in the following modules:
+//
+// * modules/olm-installing-from-operatorhub-using-cli
+// * modules/olm-installing-specific-version-cli.adoc
+// * modules/olm-operatorgroups-rbac.adoc
+// * modules/olm-operatorgroups-static.adoc
+// * modules/olm-operatorgroups-target-namespace.adoc
+// * modules/olm-policy-scoping-operator-install.adoc
+
+:_content-type: SNIPPET
+
+[WARNING]
+====
+Operator Lifecycle Manager (OLM) creates the following cluster roles for each Operator group:
+
+* `<operatorgroup_name>-admin`
+* `<operatorgroup_name>-edit`
+* `<operatorgroup_name>-view`
+
+When you manually create an Operator group from the CLI, you must specify a unique name that does not conflict with the existing cluster roles or other Operator groups on the cluster.
```

Comment:
Is "from the CLI" necessary. Will the console block duplicate names?

Reply:
Thank you for pointing that out @tmshort and talking this through with me in Slack. I have removed the phrase "from the CLI."